src/IonParserBinaryRaw.ts

```diff
@@ -317,6 +317,10 @@ export class ParserBinaryRaw {
     return Decimal._fromBigIntCoefficient(isNegative, coefficient, exponent);
   }
 
+  source(): BinarySpan {
```

Comment:
I think we should mark the `source()` methods (here and in `IonParserTextRaw`) as `private`. The `BinarySpan` and `StringSpan` classes need to be reviewed before we commit to them in the public API.

Reply:
Hi @zslayton , thanks so much for taking time to review the PR!

But question: if we mark `source()` private, how can we call this function in `BinaryReader`/`TextReader` and get position there?
Or do you suggest in `parserBinaryRaw`, we do something like:
```suggestion
  getPosition(): number {
    return this._in.postion();
  }
```
and call `this._parser.getPosition()` in `BinaryReader`/`TextReader` ?

Reply:
Cool, I will keep the first revision